packages/site/src/routes/[dictionaryId]/entries/print/PrintFieldCheckboxes.svelte

```diff
@@ -2,21 +2,23 @@
   import type { Readable } from 'svelte/store';
   import { t } from 'svelte-i18n';
   import { StandardPrintFields, type ExpandedEntry, type IPrintFields } from '@living-dictionaries/types';
+  import { convert_and_expand_entry } from '$lib/transformers/convert_and_expand_entry';
 
   export let entries: ExpandedEntry[];
   export let preferredPrintFields: Readable<IPrintFields>;
   export let showLabels: Readable<boolean>;
   export let showQrCode: Readable<boolean>;
 
   $: fieldsThatExist = Object.keys($preferredPrintFields).filter((field) => {
```

Comment:
Please revert all changes to this file, as we want the gloss field always set to true. I realized we can get typing to work here by updating this line to `$: fieldsThatExist = (Object.keys($preferredPrintFields) as (keyof IPrintFields)[]).filter((field) => {` so please do that while you're already in here.

Reply:
Somehow this comment didn't get put on the right line (12) but I think you can understand it still.